d3a_interface/constants_limits.py

```diff
@@ -122,6 +122,8 @@ class WindSettings:
         MAX_WIND_TURBINE_OUTPUT_W = 160
 
     class IAASettings:
+        GRID_FEE_TYPE = 1
+        VALID_FEE_TYPES = [1, 2]
```

Comment:
BTW, can you also add a small comment that defines which value is the constant fees and which is the percentage fees ? Thanks

Reply:
Good idea! Will do!